### DISCUSSION TITLE
Welcome to Document Viewer - CLI Discussions!

### DISCUSSION THREAD

**Discussion by Sepehrworklife** [Sepehrworklife/doc-viewer-cli#1] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by Sepehrworklife** [comment]
@SajadRafati @sarinaodr Greetings guys.
Hope you are well.
I am starting a new free and open-source project called **"Document Viewer - CLI"** as you can see.
This project is about viewing popular documents like FastAPI, Django, Flask, ... through the command line so developers don't bother themselves to search from the web or framework/library websites.
I would be glad if you guys want to contribute to this project, it's pleasure for me :)
The project structure is not been set already but I'm into it for sure, after a couple of days you can see the basis of the project.

Wish you the bests.

**Reply by sarinaodr** [reply]
Hello there
I'm so glad to hear that, and will do my best to be usefull😊

**Reply by Sepehrworklife** [reply]
> Hello there I'm so glad to hear that, and will do my best to be usefullblush

That's nice :smile: 
I will notify you as soon as I created the basis of the project.

**Comment by SajadRafati** [comment]
Hello, dear Sepehr
I am honored to be with you and participate in this project.

If there is a chance, let's meet and talk about it.

**Reply by Sepehrworklife** [reply]
Glad to hear that, I will be in touch with you very soon.

**Comment by Sepehrworklife** [comment]
@SajadRafati  @sarinaodr  
I encourage you guys to read [Typer Document](https://typer.tiangolo.com/) bc we will use it and if you have some free time to read about CLI Apps, it will be helpful.

**Reply by Sepehrworklife** [reply]
[Github API](https://docs.github.com/en/rest?apiVersion=2022-11-28) is important too, we will use it to retrieve our data from repositories documents.

**Comment by Sepehrworklife** [comment]
@sarinaodr @SajadRafati I will create many issues in the future, if any of you wanted to cooperate let me know on that specific issue so I assign that to you.
There is a label called `good first issue` like this: #2, it's good for beginners to contribute.

**Reply by Sepehrworklife** [reply]
Make sure to read [Contributing Instructions](https://github.com/Sepehrworklife/doc-viewer-cli/blob/master/COUNTRIBUTING.md) first.

**Reply by sarinaodr** [reply]
If it's good for beginners, I'm up to doing that :))))

**Reply by Sepehrworklife** [reply]
Great!
Please say something at #2 so I can assign you to that issue.
